### PR TITLE
Add define plugin

### PR DESCRIPTION
**Pull request by Raptor123471**
Plugin to grab word definitions from http://www.dictionary.com/

### CODE REVIEW

**Comment by gurumadmat** [comment]
Tested and works.
Provide some wiki info so we can merge it in staging.

**Comment by Raptor123471** [comment]
@gurumadmat I added a wiki page for it

File: hangupsbot/plugins/define.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+
+    url = "http://www.dictionary.com/browse/" + argument + "?s=t"
+ 
+    try:
+        response = urllib.request.urlopen(url)
+        html = response.read()
+     
+    except:
+           #404 caused by word not found
+           yield from bot.coro_send_message(event.conv, "No defenition for " + argument.capitalize())
```
Comment (davidlinse) [inline hangupsbot/plugins/define.py]: Typo.. should be _definition_ instead of _defenition_.
regards
~david

**Comment by Raptor123471** [comment]
@pleasantone I would consider using the spawn API but i have no idea where to start with it. Are there any examples I can reference? Also, will it work on Windows? I looked at the wiki page and I still don't seem to understand how it works.

**Comment by pleasantone** [comment]
It just runs a program and pipes the output back.
That is the API.

Should work on windows too

**Comment by gabeduke** [comment]
I already wrote a spawn for this. Is there a repository to share  them?

**Pull request closed without merging**